### ISSUE TITLE
[Topic]: Principles of Accessible Design]

### ISSUE THREAD

**Issue by lakeen** [w3c/wai-curricula#73]
Minor issues:

Introduce the terms “Perceivable”, “Operable”, “Understandable”, and “Robust” (POUR). Use the resource **Accessibility Principles** [link not found] - should be: https://www.w3.org/WAI/fundamentals/accessibility-principles/

Under Homework Ideas:

**Unfinished sentence:** Pick up at least three accessibility requirements and ask students to determine which principle they belong to. Encourage them to list at least two guidelines related to the requirement you have chosen. **For instance, use the** ...

**Comment by daniel-montalvo** [comment]
Thanks @lakeen  and @eoncins for pointing to this. Now links should work as expected both in the frozen and live drafts.

Feel free to reopen if necessary.
